Nasal/c172p.nas

```diff
@@ -704,7 +704,8 @@ setlistener("/sim/signals/fdm-initialized", func {
 
     setlistener("/engines/active-engine/cranking", func (node) {
         setprop("/engines/active-engine/external-heat/enabled", 0);
-        setprop("/sim/externalheater/enable", 0)
```

Comment:
By accident I found the missing statement terminator here,
so i decided to just fix it too